### ISSUE TITLE
Tabs - role="tabpanel" has no accessible name

### ISSUE THREAD

**Issue by mathalete** [Workday/canvas-kit#1014]
## 🐛 Bug Report

Each `role="tabpanel"` has no accessible name.

The value for `aria-labelledby` on each `role="tabpanel"` self refers to the ID of the `role="tabpanel"` itself.

## To Reproduce

Inspect the DOM and look at the DOM node value used in the ID for each `role="tabpanel"` and observe it is self-referring. 

## Expected Behavior

The ID value of the `aria-labelledby` attribute on each `role="tabpanel` needs to point to the ID of its corresponding `role=tab`.

## Link to repl or repo (highly encouraged)

This is relevant to all examples for tabs in  https://workday.github.io/canvas-kit/?path=/story/testing-react-labs-tabs--simple

## Screenshot of DOM and accessibility node for `role="tabpanel"` shown below.

![CleanShot 2021-04-01 at 14 06 16](https://user-images.githubusercontent.com/439974/113298253-89b54a80-92f3-11eb-8cb4-a6a124a32e7b.png)

**Comment by NicholasBoll** [comment]
It looks like the `tabpanel`  has an accessible name. The `role=tabpanel`'s `aria-labelledby` doesn't reference itself. The difference is subtle:
```
aria-labelledby="tab-*"
id="tabpanel-*"
```

Chrome is showing the following:
![image](https://user-images.githubusercontent.com/338257/116936024-7106c000-ac24-11eb-8937-fd290ff5da28.png)

This is according to the ARIA specifications and is covered by tests. Your screenshot looks like Safari. Is Safari doing the wrong thing?

**Comment by NicholasBoll** [comment]
I don't see any issue. It seems to be working as expected. I'm going to close unless there's a specific issue.
